recipe/meta.yaml

```diff
@@ -93,6 +93,7 @@ outputs:
         - cmake
         - ninja-base
         - pkgconfig
+        - git  # [osx or unix or aarch64]
```

Comment:
`unix` already includes `osx` and `aarch64`, see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

Reply:
@skupr-anaconda Without that entry, the build fails.